code/modules/shuttle/arrivals.dm

```diff
@@ -7,6 +7,7 @@
 	height = 15
 	dir = WEST
 	port_angle = 180
+	preferred_direction = WEST

```

Comment:
You shouldn't be setting these things in Initialize, because mappers may place it elsewhere, and I don't know why you moved the docking ports.